packages/react-test-renderer/shallow.js

```diff
@@ -0,0 +1,7 @@
+'use strict';
+
+if (process.env.NODE_ENV === 'production') {
+  throw Error('shallow renderer is not available in production mode.');
+} else {
```

Comment:
Or should we create prod versions of all of them? I don't see benefit but somebody might come up with a use case. Maybe just worth keeping in mind.